### ISSUE TITLE
New Ice-like mobs for new map

### ISSUE THREAD

**Issue by Hawk-v3** [Yawn-Wider/YWPolarisVore#3]
Some ice-like Angry fauna.
Doesn't have to be animated or anything
But anything ice-themed would be good for the new map
And maybe some friendly fauna, too,
Sprites currently being done by Harpsong (discord).

**Comment by RadiantFlash7** [comment]
Aaaa! This isn't done. We only have a single ice-mob thats not from virgo. I require mooore.

**Comment by SplinterGP** [comment]
Need suggestiuns! Revive

**Comment by SplinterGP** [comment]
A mob that can somehow use a phoron crystal or something to heat themselves on the cold, making them able to survive at the blizzard(they'd have to stand still for that however) and would heat station if would get inside, Possible for science to use them as Improvised heaters.

**Comment by izac112** [comment]
I mean when it comes to blizzards all the mobs should survive it its just the way the blizzard works right now aka like an ash storm it just slaps on damage unless you got the flag that makes you completly fire immune

**Comment by SplinterGP** [comment]
yeah uh, just for context on how they are able to withstand, another idea too would be some mole or something that could hide at snow when in a blizzard and stay for a time, if someone would step on the "mole trap" they could get hit or something.

**Comment by izac112** [comment]
Sounds neat

**Comment by SplinterGP** [comment]
also something that might want to be done is for blizzard to cover uncovered tiles in snow, atleast the ones people shoveled.

**Comment by izac112** [comment]
Thats something we wanted to add but never got around to doing, like  i said before right now its pretty much just an ash storm with a diffrent name

**Comment by SplinterGP** [comment]
aye, i'll took a look into.

**Comment by izac112** [comment]
Thank you
